File: demo/TestDemo/src/main/java/com/lu/magic/TestMainActivity.java
package com.lu.magic;

import android.os.Bundle;
import android.os.VibratorManager;

import androidx.annotation.Nullable;

import com.blankj.utilcode.util.VibrateUtils;
import com.lu.magic.testdemo.databinding.LayoutMainBinding;
import com.lu.magic.ui.BaseActivity;

public class TestMainActivity extends BaseActivity {
    private LayoutMainBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = LayoutMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.btnClickVibrate.setOnClickListener((v) -> {
            VibrateUtils.vibrate(75L);
        });
    }
}

File: demo/TestDemo/src/main/java/com/lu/magic/TestMainActivity.kt
package com.lu.magic

import android.os.Bundle
import android.view.View
import com.blankj.utilcode.util.VibrateUtils
import com.lu.magic.config.ConfigUtil
import com.lu.magic.frame.xp.BRPreference
import com.lu.magic.testdemo.databinding.LayoutMainBinding
import com.lu.magic.ui.BaseActivity

class TestMainActivity : BaseActivity() {
    private lateinit var sp: BRPreference
    private var binding: LayoutMainBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ConfigUtil.initWithReadable(this)
        binding = LayoutMainBinding.inflate(layoutInflater)
        setContentView(binding!!.root)
        binding!!.btnClickVibrate.setOnClickListener { v: View? -> VibrateUtils.vibrate(75L) }

    }

    override fun onResume() {
        super.onResume()

    }

}